multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -1,5 +1,7 @@
 from typing import List, Optional
 
+from autogluon.common.utils.try_import import try_import_ray
```

Comment:
I think no need to add `try_import_ray` in presets.py. The one in predictor.py can always catch the error for HPO.